src/Catalyst.Core.Modules.P2P.Discovery.Hastings/HastingsDiscovery.cs

```diff
@@ -249,10 +239,7 @@ public async Task DiscoveryAsync()
         /// </summary>
         protected void WalkForward()
         {
-            if (!StepProposal.HasValidCandidate())
-            {
-                return;
-            }
+            if (!StepProposal.HasValidCandidate()) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)